### ISSUE TITLE
Make GUI/dashboard.py an application

### ISSUE THREAD

**Issue by achoudh5** [achoudh5/Manipulating_Excel_Sheet#23]
Currently, the `dashboard.py` file under GUI folder is a mere code that someone needs to run on their cli. 

**Problem statement:-**

- If a user who doesn't know how to use cli might never run this code under `dashboard.py`. We have to make sure they do use this wonderful product. (**Hint:** Can we make this tkinter GUI an application?)

- Find a best way for users to download this application, do they need to git clone only? can there be any other way of doing it?

- We need to make sure all the requirements are met when they install application ie all necessary packages and everything shouldn't be an issue irrespective of their OS (Maybe docker)

**Comment by godspeed5** [comment]
I can work on this

**Comment by godspeed5** [comment]
You don't seem to have requirements.txt. That would fix issue 3?

**Comment by achoudh5** [comment]
Yes, @godspeed5 that will help but it's in development. I added my sample later and want people to take initiative  and find missing links (will be a good practise for them). 

Question:- What if the project was like this, can you find which modules were used in `dashboard.py` 
succinctly? What if I don't want to check errors on my cli and then make a requirements.txt, can I read a script and just find out the missing modules and create a requirements.txt? Let me know your thoughts and I'll create a separate issue for this :)

PS:- Meanwhile, If you want to make a blueprint for this application  and import `dashboard.py` in your code, you should be all set to work on this PR!
